.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+# https://github.com/changesets/action
+name: Release
+
+on:
+  push:
+    branches:
+      - production
+
+env:
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Any reason the `NPM_TOKEN` variable is set here and not within the changesets step? I don't think any other steps need it.

Reply:
Not a particular reason and I have moved to the `changeset` block